gatsby-config.js

```diff
@@ -47,8 +47,8 @@ module.exports = {
     {
       resolve: `gatsby-plugin-manifest`,
       options: {
-        name: `Barry McGee`,
-        short_name: `barrymcgee`,
+        name: `Ciaran McKenna`,
+        short_name: `ciaranmckenna`,
         start_url: `/`,
         background_color: `#508b62`,
         theme_color: `#508b62`,
```

Comment:
Looks good.

While you're here you may also want to change the theme color in this config file to match the theme color set in your CSS. 

https://github.com/ciaranmckenna/ciaranmckenna.com/blob/master/src/styles/theme.scss